ext/net/lib.deno_net.d.ts

```diff
@@ -76,6 +76,9 @@ declare namespace Deno {
      * You should show the message like `server running on localhost:8080` instead of
      * `server running on 0.0.0.0:8080` if your program supports Windows. */
     hostname?: string;
+    reuseAddress?: boolean;
+    /** SO_REUSEADDR option for socket.
+     * If not specified, defaults to true in Linux, false in Windows. */
```

Comment:
This docstring should be above `reuseAddress` field

Reply:
Docstring moved in the correct position